pandas/io/formats/format.py

```diff
@@ -1513,20 +1588,30 @@ def _format_strings(self):
 
 
 class Timedelta64Formatter(GenericArrayFormatter):
-    def __init__(self, values, nat_rep="NaT", box=False, **kwargs):
+    def __init__(
+        self,
+        values: Union[ndarray, TimedeltaIndex],
+        nat_rep: str = "NaT",
+        box: bool = False,
+        **kwargs
+    ):
         super().__init__(values, **kwargs)
         self.nat_rep = nat_rep
         self.box = box
 
-    def _format_strings(self):
+    def _format_strings(self) -> ndarray:
         formatter = self.formatter or _get_format_timedelta64(
             self.values, nat_rep=self.nat_rep, box=self.box
         )
         fmt_values = np.array([formatter(x) for x in self.values])
         return fmt_values
 
 
-def _get_format_timedelta64(values, nat_rep="NaT", box=False):
+def _get_format_timedelta64(
+    values: Union[ndarray, TimedeltaIndex, TimedeltaArray],
```

Comment:
really?

Reply:
similiarly, _get_format_timedelta64 is imported and used by `core\arrays\timedeltas.py` and `core\indexes\timedeltas.py`